File: py_src/step12/10814.py
n = int(input())
member = list()

for _ in range(n):
    age, name = map(str, input().split())
    member.append((int(age), name))

member.sort(key = lambda x : x[0])

for i in range(n):
    print(member[i][0], member[i][1])

File: baekjoon/js/10814.js
const fs = require("fs");
const buf = fs.readFileSync(0, "utf-8").toString().trim();

const [, ...inputs] = buf.split("\n");

const members = inputs.map((input) => input.split(" "));

members.sort((a, b) => a[0] - b[0]);

console.log(members.map((member) => member.join(" ")).join("\n"));
